system-test/bigtable.js

```diff
@@ -0,0 +1,492 @@
+/*!
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var assert = require('assert');
+var async = require('async');
+var uuid = require('node-uuid');
+
+var Row = require('../lib/bigtable/row.js');
+
+var env = require('./env.js');
+var gcloud = require('../lib/index.js')(env);
+
+var clusterName = process.env.BIGTABLE_CLUSTER;
```

Comment:
Nope, I'll update it. Also, since we separated the zone from the cluster property, that would mean we'd have to set a variable for both the zone and the cluster now?